deploy/crds/operators_v1_catalogsourceconfig_crd.yaml

```diff
@@ -39,6 +39,9 @@ spec:
           - targetNamespace
           - packages
           properties:
+            source:
+              type: string
+              description: The name of the OperatorSource that the operators originate from
```

Comment:
We should call out that this is optional.

Reply:
In the description or is there a specific tag that specifies an optional field?

Reply:
Please look into this

Reply:
I reviewed the [OpenAPI-specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#fixed-fields-11) and found that the `required` field is meant to specify whether or not a field is optional. The absense of this field suggests that one does not need to set it.

Personally, I believe that updating the description to specify the following is the correct move:
```
source:
              type: string
              description: An optional field that specifies the name of an OperatorSource that the operators must originate from
```

Reply:
That is what I was asking for. I am also wondering if we should even add this as a optional item. I feel this will cause confusion. I think we should go down the route of clearly separating the the two versions. `v1` you cannot specify an `OperatorSource`. `v2 specifying an `OperatorSource` is mandatory.

Reply:
Let's move towards v2.